bench-tps/src/bench.rs

```diff
@@ -1160,4 +1184,60 @@ mod tests {
         }
         withdraw_durable_nonce_accounts(client, &authority_keypairs, &nonce_keypairs)
     }
+
+    #[test]
+    fn test_bench_tps_key_chunks_new() {
+        let num_keypairs = 16;
+        let chunk_size = 4;
+        let keypairs = (0..num_keypairs)
+            .map(|_| Keypair::new())
+            .collect::<Vec<_>>();
```

Comment:
nit: `std::iter::repeat_with(|| Keypair::new()).take(num_keypairs).collect::<Vec<_>>()` (ditto for the next test)

Reply:
What's the benefit of that over what I've got?

Reply:
^ I ended up doing this as it looks a bit nicer actually after sitting with it for a few minutes.